readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -97,6 +97,7 @@ sphinx:
   # The builder type for the sphinx documentation
   # Default: 'html'
   builder: enum('html', 'htmldir', 'singlehtml', required=False)
+  builder: enum('html', 'dirhtml', 'singlehtml', required=False)
```

Comment:
here you should modify the line above adding the new type in the enum (here we need to keep both, old and new value).